setup.py

```diff
@@ -8,15 +19,24 @@
     'Topic :: Scientific/Engineering'
 ]
 
-setup(
-    name='fastdtw',
-    version='0.2.2',
-    author='Kazuaki Tanida',
-    url='https://github.com/slaypni/fastdtw',
-    description='Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity.',
-    license='MIT',
-    keywords=['dtw'],
-    py_modules=['fastdtw'],
-    install_requires=['six'],
-    classifiers=classifiers
-)
+kwargs = {
+    'name': 'fastdtw',
+    'version': '0.2.2',
+    'author': 'Kazuaki Tanida',
+    'url': 'https://github.com/slaypni/fastdtw',
+    'description': 'Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity.',
+    'license': 'MIT',
+    'keywords': ['dtw'],
+    'install_requires': ['six'],
+    'packages': find_packages(),
+    'ext_modules':  extensions,
+    'py_modules': ['fastdtw'],
+    'classifiers': classifiers
+}
```

Comment:
```py
    `test_suite`: `fastdtw.tests`,
```
So that `python setup.py test` works.

Which reminds me. Please add a test that shows the two implementations return _the same results_. Perhaps also print a small performance benchmark.